portfolio/src/main/webapp/script.js

```diff
@@ -32,10 +32,17 @@ function changeCatPhoto() {
 }
 
 /* Displays all comments left on page. */
-async function getComments() {
-  const response = await fetch('/data').then(response => response.json())
+async function getComments(max) {
+  document.getElementById('comments-section').innerHTML = "";
```

Comment:
nit: string literals should use single quotes

https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes